silkrpc/json/types.cpp

```diff
@@ -113,6 +113,20 @@ void from_json(const nlohmann::json& json, uint256& ui256) {
 
 namespace silkworm {
 
+void from_json(const nlohmann::json& json, std::vector<silkworm::AccessListEntry>& al) {
```

Comment:
Type serializers for STL containers are not necessary in `nlohmann`: it should be sufficient to define the serializer for the element type, i.e. `AccessListEntry` in this case